cities/cities.json

```diff
@@ -87,10 +87,6 @@
         "id": "hilden",
         "label": "Hilden"
     },
-    "kaiserslautern": {
```

Comment:
Further, please check the other files (_cities.json_, _README.md_): in your current commit you delete entries for Kaiserslautern and Zwickau. Please undo the deletion!
In _cities.json_ the entry for Trier is missing.